File: lecture001/oddeven.java
import java.util.Scanner;
//verified
public class oddeven{
    public static void main(String[] args){
        Scanner scn=new Scanner(System.in);

        System.out.print("Enter no :");
        int a=scn.nextInt();

        if(a%2==0){
            System.out.print("no is even");
        }
        else{
            System.out.print("no is odd");
        }
    }

}

File: lecture001/oddeven.cpp
#include <iostream>
using namespace std;
int main()
{
    int a;
    cout << "Enter a numbers : ";
    cin >> a;
    if (a % 2 == 0)
    {
        cout << a << " is even";
    }
    else
    {
        cout << a << " is odd";
    }
}